packages/plugin-ext/src/plugin/node/debug/plugin-debug-adapter-starter.ts

```diff
@@ -16,6 +16,7 @@
 
 import * as net from 'net';
 import * as theia from '@theia/plugin';
+import { environment } from '@theia/core/shared/@theia/application-package/lib/environment';
```

Comment:
IIUC the `src/plugin` folder is meant to be plugin-host code. Is that coupling to `@theia/application-package` causing any issue to keep that process lightweight?

Reply:
I think it should not be a problem.
cc @benoitf

Reply:
I used `is-electron` directly instead, it is lighter and precisely what we want.